bot/cogs/filtering.py

```diff
@@ -176,6 +191,20 @@ def mod_log(self) -> ModLog:
                             if _filter["user_notification"]:
                                 await self.notify_member(msg.author, _filter["notification_msg"], msg.channel)
 
+                        # If the message is classed as offensive, we store it in the site db and
+                        # it will be deleted it after one week.
+                        if _filter["schedule_deletion"]:
+                            delete_date = (msg.created_at + OFFENSIVE_MSG_DELETE_TIME).isoformat()
+                            data = {
+                                'id': msg.id,
+                                'channel_id': msg.channel.id,
+                                'delete_date': delete_date
+                            }
+
+                            await self.bot.api_client.post('bot/offensive-messages', json=data)
+                            self.schedule_task(self.bot.loop, msg.id, data)
```

Comment:
`schedule_task` no longer takes the loop as an argument. Go through the cog and remove the loop argument from all calls to this function.